backend/djangoindia/api/serializers/event.py

```diff
@@ -55,6 +55,23 @@ class EventRegistrationSerializer(serializers.Serializer):
     other_links = serializers.URLField(required=False, allow_blank=True)
     rsvp = serializers.BooleanField(default=False)
     include_in_attendee_list = serializers.BooleanField(default=False)
+    first_time_attendee = serializers.BooleanField(default= True)
+    attendee_type = serializers.ChoiceField(choices=EventRegistration.AttendeeType)
+    
+    class EventAttendeeSerializer(serializers.ModelSerializer):

```

Comment:
why do we need an inner class `ModelSerializer`  that doesn't connect to any model?

Reply:
<img width="211" alt="khi" src="https://github.com/user-attachments/assets/a209898e-a4d7-4d11-b958-6e7573afe441">
Bhaiya I am not able to change it to different attende_type from admin panel. Can u please help me with that

Reply:
@salty-ivy